src/Sylius/Bundle/AddressingBundle/SyliusAddressingBundle.php

```diff
@@ -25,24 +27,20 @@ class SyliusAddressingBundle extends AbstractResourceBundle
     /**
      * {@inheritdoc}
      */
-    public static function getSupportedDrivers()
+    public function build(ContainerBuilder $container)
     {
-        return array(
-            SyliusResourceBundle::DRIVER_DOCTRINE_ORM,
-        );
+        parent::build($container);
+
+        $container->addCompilerPass(new ServicesPass());
     }
 
     /**
      * {@inheritdoc}
      */
-    protected function getModelInterfaces()
+    public static function getSupportedDrivers()
```

Comment:
Not sure it is usefull... Configuration should should be aware of the available drivers and restrict them.